tests/user/test_user_auth exceptions.py

```diff
@@ -80,7 +80,7 @@
         ),
         (
             InvalidLambdaEventInput,
-            {"statusCode": 500, "body": "Internal Server Error"},
+            {"statusCode": 400, "body": "Invalid authentication input"},
```

Comment:
Is this desired? That could be either 500 for misconfigured integration between APIGW and Lambda, or 400 due to the client passing wrong input data.

I suggest not mapping this exception directly to a status code but differentiating it through other exceptions.

Reply:
For now I just rollback to 500 since any other exceptions not mapped return the same status code.